overlord/configstate/configcore/backlight_test.go

```diff
@@ -41,13 +41,12 @@ func (s *backlightSuite) SetUpTest(c *C) {
 
 	s.systemctlArgs = nil
 
-	dirs.SetRootDir(c.MkDir())
 	err := os.MkdirAll(filepath.Join(dirs.GlobalRootDir, "/etc/"), 0755)
 	c.Assert(err, IsNil)
 }
 
 func (s *backlightSuite) TearDownTest(c *C) {
-	dirs.SetRootDir("/")
+	s.configcoreSuite.TearDownTest(c)
```

Comment:
If the only call of "TearDownTest" is now to call the parent you can just remove the entire function, go will automatically call s.configcoreSuite.TearDownTest(c) if it's not overridden. Same in some more places in this PR, just removing the TearDownTest() is fine.

Reply:
Thanks, you're right! Done.